tool/travis.sh

```diff
@@ -22,7 +22,6 @@ function run_coveralls {
 pushd build_test
 pub upgrade
 dartanalyzer --fatal-warnings lib/build_test.dart
-pub run test
```

Comment:
It does cause it to fail. It prints
"No test files were passed and the default "test/" directory doesn't exist." and usage and returns a non-zero exit code.